package.json

```diff
@@ -16,13 +16,14 @@
   "homepage": "https://github.com/eclipse-cdt-cloud/cdt-gdb-vscode#readme",
   "main": "./out/extension",
   "engines": {
-    "vscode": "^1.76.0"
+    "vscode": "^1.78.0"
```

Comment:
Normally I would want version change like this to be split out - but since there is a dependency here on https://github.com/microsoft/vscode/issues/175945 I think it is ok this time.

Reply:
Is this version bump still needed since you aren't using new feature? Lets split out the version bump to a new PR.